libs/base/src/lib/context/NetworkProvider.tsx

```diff
@@ -170,20 +170,28 @@ const ETH_MAINNET: EthereumMainnet = {
   gasStationEndpoint: 'https://gas.mycryptoapi.com/',
   gqlEndpoints: {
     protocol: [
-      graphHostedEndpoint('mstable', 'mstable-protocol'),
-      graphMainnetEndpoint('0x26cf67040678eb0f5654c9cbaad78dc1694cbafa', 0, process.env.NX_PROTOCOL_SUBGRAPH_API_KEY as string),
+      'https://api.studio.thegraph.com/query/32034/mstable-protocol-2/v1.0.1',
```

Comment:
do we need to keep this endpoint?

Reply:
this is added for now to avoid high cost on calling published subgraphs